src/scripts/crowdin/reports/reportsHelpers.ts

```diff
@@ -25,22 +24,45 @@ export async function fetchTranslationCostsReport(
 
   // Todo: Remove ts-ignore when this PR gets merged
   // https://github.com/crowdin/crowdin-api-client-js/pull/282
-  const schema: ReportsModel.TranslationCostSchema = {
+  // const schema: ReportsModel.TranslationCostSchema = {
+  const schema: ReportsModel.TranslationCostsPostEndingSchema = {
```

Comment:
this doesn't seem the PR to include this change. do we need to test something? not sure what this change is doing